drivers/serial/uart_ite_it8xxx2.c

```diff
@@ -15,8 +15,9 @@
 #include <pm/policy.h>
 #include <soc.h>
 
+#define LOG_LEVEL CONFIG_UART_LOG_LEVEL
 #include <logging/log.h>
-LOG_MODULE_REGISTER(uart_ite_it8xxx2, LOG_LEVEL_ERR);
+LOG_MODULE_REGISTER(uart_ite_it8xxx2);
```

Comment:
Same here. You are actually reverting from the two-argument `LOG_MODULE_REGISTER()` to a one-argument version.